source/common/common/logger.h

```diff
@@ -22,6 +22,7 @@ namespace Logger {
 // clang-format off
 #define ALL_LOGGER_IDS(FUNCTION) \
   FUNCTION(admin)                \
+  FUNCTION(aws)                  \
```

Comment:
can you add a TODO to figure out some way for extensions to register new logger IDs?

Reply:
done

Reply:
I haven't been looking at these PRs but just wanted to note that in #5580 we decided to not add aws: https://github.com/envoyproxy/envoy/pull/5580#discussion_r248214667

So I'm surprised this got back in. @ivitjuk It would make sense to have the sigv4 signer use it again since the logging the signer does it very AWS-specific.

Reply:
Makes sens to me. I'll make a PR to switch signer to the AWS logger.